Gruntfile.js

```diff
@@ -144,8 +146,12 @@ module.exports = function(grunt) {
 
     grunt.config.set("gh-release.release-notes", releaseNotes);
 
-    grunt.file.write(dest, "" + header + contents);
-    grunt.log.ok("Changelog updated, and release notes extracted.");
+    if (dryRun) {
+      grunt.log.ok('changelog (dry): Prepending header to ' + dest);
```

Comment:
In that case, I think it's fine to update Changelog but not commit it - that way, I can just git checkout -- . once I'm done eyeballing what the script spits out.